src/types/utilities.ts

```diff
@@ -151,7 +163,11 @@ export function isPropertyReadonlyInType(
 	return false;
 }
 
-/** Returns true for `Object.defineProperty(o, 'prop', {value, writable: false})` and  `Object.defineProperty(o, 'prop', {get: () => 1})`*/
+/**
+ * Returns true for `Object.defineProperty(o, 'prop', {value, writable: false})` and  `Object.defineProperty(o, 'prop', {get: () => 1})`
```

Comment:
[Nitpick] Just noticed this 😄 

```suggestion
 * Returns true for `Object.defineProperty(o, 'prop', {value, writable: false})` and `Object.defineProperty(o, 'prop', {get: () => 1})`
```